addon/number.js

```diff
@@ -30,16 +30,20 @@ import validationError from 'ember-validators/utils/validation-error';
 export default function validateNumber(value, options) {
   let numValue = Number(value);
   let optionKeys = Object.keys(options);
-  let { allowBlank, allowNone = true, allowString, integer } = getProperties(options, ['allowBlank', 'allowNone', 'allowString', 'integer']);
```

Comment:
@bgentry by removing `= true` here you are changing the default behavior of the validator.

Before: 

```js
validateNumber(null, {}) // --> true
```

After:

```js
validateNumber(null, {}) // --> false
```

This will be a breaking change for both ember-cp-validations & ember-changeset-validations + any other addon that consumes this package.

Reply:
Ok, but just to be clear I believe that leaving the default might also be considered a breaking change in some cases. The behavior of existing users' validations will change such that they will no longer see validation errors where they might have previously.

If you assume the docs are your official API contract with users, then that would be acceptable.

I'll reinstate the default.

Reply:
> The behavior of existing users' validations will change such that they will no longer see validation errors where they might have previously.

Ah, I see what you mean. That is fine, since that's the behavior it should have been. I'll tag this as a minor release and point out the change.

> I'll reinstate the default.

Thank you!